src/mscorlib/src/System/BitConverter.cs

```diff
@@ -339,21 +339,25 @@ public static bool ToBoolean(byte[] value, int startIndex)
             return value[startIndex] != 0;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)] // This should compile down to nothing or roughly 1 instruction, depending on the register holding value
```

Comment:
unless inlined, in which case the value should already be in a register and requires no translation (which is what I am seeing). I'll post the disasm tomorrow